src/data/nat/prime.lean

```diff
@@ -540,6 +540,10 @@ lemma mem_factors_iff_dvd {n p : ℕ} (hn : 0 < n) (hp : prime p) : p ∈ factor
 ⟨λ h, prod_factors hn ▸ list.dvd_prod h,
  λ h, mem_list_primes_of_dvd_prod hp (@mem_factors n) ((prod_factors hn).symm ▸ h)⟩
 
+lemma mem_factors' {n p} (hn : 0 < n) : p ∈ factors n ↔ prime p ∧ p ∣ n :=
```

Comment:
I think that this should be `nat.mem_factors`, and the current `nat.mem_factors` should be called `nat.prime_of_mem_factors`.